docs/content/concepts/assets/asset-checks.mdx

```diff
@@ -285,3 +285,9 @@ defs = Definitions(
     schedules=[basic_schedule],
 )
 ```
+
+---
+
+## Scaling limits
```

Comment:
`## Limitations`?

Should this be its own section? Could also add a section about defining multiple checks on one asset

Reply:
+1 to it being **Limitations**. It gives us room to add more if we need to, and this is consistent with other docs/features that also have limitation sections.

Also, I feel pretty strongly about having limitation sections near the top of pages versus the bottom. As a user, I'd prefer to know what the limits of things are upfront. I've been reworking concept pages to follow this structure:

```
- Title/overview
- Prerequisites (if any)
- Limitations (if any)
- Relevant APIs
- Defining [concept]
- ...
```

Open to thoughts/feedback on this!

Reply:
I like the idea of calling it Limitations and being consistent with other docs.

I'm hesitant about putting it near the top because I think the limitations we're included here will be irrelevant information to the vast majority of users. You'd need to be using the feature in a very weird and unexpected way to actually hit this limitation.

Reply:
Other than the limitation with the UI, is there anything else that we know of that would be a limitation? Otherwise could we just embed this info in a callout in the Executing > UI section?

Reply:
It doesn't quite fit there since it's about how many checks you can define. We could call out that we don't have partition support as a limitation.

Reply:
updated